openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -9,6 +9,11 @@
     'invoice_validation_wkfl': 'account_invoice_validation_workflow',
     # OCA/server-tools
     'audittrail': 'auditlog',
+    # OCA/bank-statement-import
+    'account_banking': 'account_bank_statement_import',
+    'account_banking_camt': 'bank_statement_parse_camt',
```

Comment:
right, that's still in PR state: https://github.com/OCA/bank-statement-import/pull/15 - the problem is that all those PRs are stuck because we want to wait how Laurent's PR to 9.0 is integrated, as those modules are supposed to be as close to 9.0 as possible...